site/src/modules/provisioners/ProvisionerAlert.tsx

```diff
@@ -0,0 +1,97 @@
+import AlertTitle from "@mui/material/AlertTitle";
+import { Alert, type AlertColor } from "components/Alert/Alert";
+import { AlertDetail } from "components/Alert/Alert";
+import { Stack } from "components/Stack/Stack";
+import { ProvisionerTag } from "modules/provisioners/ProvisionerTag";
+import type { FC } from "react";
+
+interface ProvisionerAlertProps {
+	matchingProvisioners: number | undefined;
+	availableProvisioners: number | undefined;
+	tags: Record<string, string>;
+}
+
+export const ProvisionerAlert: FC<ProvisionerAlertProps> = ({
+	matchingProvisioners,
+	availableProvisioners,
+	tags,
+}) => {
+	let title: string;
+	let detail: string;
+	switch (true) {
+		case matchingProvisioners === 0:
+			title = "Provisioning Cannot Proceed";
+			detail =
+				"There are no provisioners that accept the required tags. Please contact your administrator. Once a compatible provisioner becomes available, provisioning will continue.";
+			break;
+		case availableProvisioners === 0:
+			title = "Provisioning Delayed";
+			detail =
+				"Provisioners that accept the required tags are currently anavailable. This may delay your build. Please contact your administrator if your build does not complete.";
```

Comment:
```suggestion
				"Provisioners that accept the required tags are currently unavailable. This may delay your build. Please contact your administrator if your build does not complete.";
```